### PR TITLE
Conda-package

### PR DESCRIPTION
**Pull request by claudiushaag**
- add conda installation hints
- fix gitignore with sphere hdf5 file
    - closes https://github.com/conda-forge/fans-feedstock/issues/1

Also refer to https://github.com/conda-forge/staged-recipes/pull/28099

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,18 @@ Install FANS (system-wide) using the following options:
     apt install packages/fans-dev_<version>_<architecture>.deb
     ```
 
+### Install using Conda
```
Comment (sanathkeshav) [inline README.md:147]:
```suggestion
### Install using Conda

```

**Review by sanathkeshav — COMMENTED** [review]
Check sugegstion

**Review by sanathkeshav — COMMENTED** [review]
try 2

File: README.md [hunk 2]
```diff
@@ -144,6 +144,20 @@ Install FANS (system-wide) using the following options:
...
+
+FANS is also available as a conda-package on [conda-forge](https://anaconda.org/conda-forge/fans). No dependencies have to be manually installed for it to workl.
+It can be installed via
+
+```sh
+
+conda install fans
+exposing the executable `FANS`.
+
 ## Input File Format
```
Comment (sanathkeshav) [inline README.md:161]:
```suggestion
## Input File Format {#input-file-format}
```

**Review by sanathkeshav — COMMENTED** [review]
last try

**Review by IshaanDesai — APPROVED** [review]
Looks good :+1:

**Pull request merged**